src/main/java/com/google/sps/servlets/ReviewsServlet.java

```diff
@@ -53,17 +54,21 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) {
 
       // Create review entity and get its key.
       Entity reviewEntity = new Entity("Review");
-      reviewEntity.setProperty("userID", userID);
-      reviewEntity.setProperty("comment", comment);
-      reviewEntity.setProperty("rating", rating);
-      reviewEntity.setProperty("dateTime", dateTime);
+      reviewEntity.setProperty(REVIEW_USERID, userID);
+      reviewEntity.setProperty(REVIEW_COMMENT, comment);
+      reviewEntity.setProperty(REVIEW_RATING, rating);
+      reviewEntity.setProperty(REVIEW_DATETIME, dateTime);
       datastore.put(reviewEntity);
       Key reviewKey = reviewEntity.getKey();
 
       // Append key to its Business entity.
       reviewsKeyList.add(reviewKey);
       businessEntity.setProperty(BUSINESS_REVIEWS, gson.toJson(reviewsKeyList));
```

Comment:
We settled on the key list as a List<String> on the Business class, you should have `reviewsKeyList` as a `List<String>` and store it in the entity that way.